packages/contracts/contracts/proofVerifiers/BaseProofVerifier.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IVerifier} from '../interfaces/IVerifier.sol';
+
+contract BaseProofVerifier {
+    IVerifier verifier;
+
+    uint256 public constant SNARK_SCALAR_FIELD =
+        21888242871839275222246405745257275088548364400416034343698204186575808495617;
+
+    struct EpochKeySignals {
+        bool revealNonce;
+        uint8 nonce;
+        uint48 epoch;
+        uint160 attesterId;
+        uint256 stateTreeRoot;
+        uint256 epochKey;
+        uint256 data;
+    }
+
+    struct ReputationSignals {
+        bool proveGraffiti;
+        bool proveMinRep;
+        bool proveMaxRep;
+        bool proveZeroRep;
+        bool revealNonce;
+        uint8 nonce;
+        uint48 epoch;
+        uint160 attesterId;
+        uint256 stateTreeRoot;
+        uint256 epochKey;
+        uint256 graffitiPreImage;
```

Comment:
there is no `graffitiPreImage` anymore (just graffiti itself)
https://github.com/Unirep/Unirep/pull/422

Reply:
@chndr1 This signal still exists it just needs to be renamed to `graffiti`

Reply:
whoops lol. renamed